### PR TITLE
[SDPSUP-2438] fix v-html ssr error

### PR DESCRIPTION
**Pull request by tim-yao**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPSUP-2438

### Changed

1.  Changed `p` element to `div` for `v-html`. Because nest `p` element is invalid HTML. https://github.com/nuxt/nuxt.js/issues/1523#issuecomment-325884055

### CODE REVIEW

**Review by MdNadimHossain — APPROVED** [review]

File: packages/components/Molecules/Card/CardHonourRoll.vue [hunk 1]
```diff
@@ -9,7 +9,7 @@
         <span v-if="inductionYear && category" class="rpl-card-honour-roll__separator">|</span>
         <span v-if="category">{{ category }}</span>
       </p>
-      <p class="rpl-card-honour-roll__summary" v-if="summary" v-html="summary" />
+      <div class="rpl-card-honour-roll__summary" v-if="summary" v-html="summary"></div>
```
Comment (rubyllamadora) [inline packages/components/Molecules/Card/CardHonourRoll.vue:12]: Or just render it like `<p...>{{ summary }}</p>`?

**Review by rubyllamadora — APPROVED** [review]

File: packages/components/Molecules/Card/CardHonourRoll.vue (hunk 1 above)
Comment (tim-yao) [reply packages/components/Molecules/Card/CardHonourRoll.vue:12]: @rubyllamadora `<p...>{{ summary }}</p>` won't render HTML, it will be encoded.

Comment (rubyllamadora) [reply packages/components/Molecules/Card/CardHonourRoll.vue:12]: Got it, there's html inside. I thought it was pure text. Summary field in BE must be using wysiwyg

Comment (tim-yao) [reply packages/components/Molecules/Card/CardHonourRoll.vue:12]: > Got it, there's html inside. I thought it was pure text. Summary field in BE must be using wysiwyg

Yes

**Pull request merged**